frontend/src/metabase/lib/formatting.js

```diff
@@ -930,13 +930,20 @@ export function conjunct(list: string[], conjunction: string) {
 }
 
 export function duration(milliseconds: number) {
-  if (milliseconds < 60000) {
-    const seconds = Math.round(milliseconds / 1000);
-    return ngettext(msgid`${seconds} second`, `${seconds} seconds`, seconds);
-  } else {
-    const minutes = Math.round(milliseconds / 1000 / 60);
+  const SECOND = 1000;
+  const MINUTE = 60 * SECOND;
+  const HOUR = 60 * MINUTE;
```

Comment:
Yeah, better.

At the same time, looks to me these constants for time can all live in one place.